src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,383 @@ tree_item_index: 1100
 
 {{! excerpt}}
 The [Nuxeo Aspera](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-aspera) connector
-enables users to download/upload binaries with Aspera.
+enables users to download/upload binaries with Aspera.  It is designed to be used with **Aspera On Cloud** intances, however, if you use a **self-hosted Aspera** instance, it is still possible you will be able to use the Nuxeo Aspera Connector.  Contact your Aspera team for more information and assistance.
 {{! /excerpt}}
 
-## Requirements
+## Version 3.0.0
+### Requirements
+
+- Aspera Desktop Client - [latest version].
+- Nuxeo Server LTS 2019 (10.10, HF 31, at least) with access to AWS S3 Storage along with the [Amazon S3 Online Storage plugin](https://connect.nuxeo.com/nuxeo/site/marketplace/package/amazon-s3-online-storage?version=11.2.13)
+- IBM Aspera on Cloud Subscription (see note above about Aspera self-hosted)
+
+### Installation
+
+Installation is made of two steps:
+
+1.  Install the [Nuxeo Package](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-aspera) available from the marketplace.
+2.  Install the [Aspera desktop client].
```

Comment:
is the idea to put a link here?

Reply:
@andreacornaglia yes, on the 2.0.x doc, there is a dead link.  I need to locate a link to the connector that works and validate the version works with our connector.

Reply:
The Aspera desktop client is required but we don't recommend to do it outside our scope, or do we?
If you don't have it, you'll be prompted on the first time you try to use our connector/integration.

Also, the browser extension is also needed and we are not referring that. Guess we should decide what direction we want to take: state those things or document the wizard